.github/snippet-bot.yml

```diff
@@ -1,4 +1,5 @@
-# Copyright 2022 Google LLC
+
+# Copyright 2021 Google LLC
```

Comment:
Question:
Should we use `2022` instead of `2021`?

Reply:
oof yes- ill change the ones that I added in for this PR, should I change the ones that are dated previous (for ex 2018?)

Reply:
Note: we're not supposed to modify those years (will link internal source)